### DISCUSSION TITLE
osu crashes sometimes when filtering by star rating at the beatmap selection screen

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#16210] [Q&A] [answered]
i just tried to search for ``stars>=5`` and suddenly osu threw an error but this doesn't seem to happen very frequently.

[runtime.log](https://github.com/ppy/osu/files/7760966/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7760967/updater.log)
[database.log](https://github.com/ppy/osu/files/7760968/database.log)
[network.log](https://github.com/ppy/osu/files/7760969/network.log)
[performance.log](https://github.com/ppy/osu/files/7760970/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/7760971/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7760972/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/7760973/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/7760974/performance-update.log)

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/16211.
